R/dfm.R

```diff
@@ -182,6 +200,11 @@ dfm.tokens <- function(x,
 
     check_dots(otherargs, method = "dfm")
 
+    remove_padding <- check_logical(remove_padding)
+    if (remove_padding) {
+        x <- tokens_remove(x, "")
```

Comment:
Adding `valuetype = "fixed"` will disable auto-detection of glob in `pattern2id()` but I admit that the difference is very small.

Reply:
More importantly, it is faster to remove padding after forming a DFM.

```r
> toks <- tokens(data_corpus_inaugural, remove_punct = TRUE, padding = TRUE)
> microbenchmark::microbenchmark(
+   dfm(tokens_remove(toks, "")),
+   dfm_remove(dfm(toks), "")
+ )
Unit: milliseconds
                         expr     min       lq     mean   median       uq      max neval
 dfm(tokens_remove(toks, "")) 73.2486 94.10910 151.1463 126.0674 172.1803 399.8985   100
    dfm_remove(dfm(toks), "") 65.8952 87.95525 122.9699 101.7932 134.7257 330.3436   100
```

Reply:
I'm getting very similar results for all three options below. Looks like the `remove_padding = TRUE` is about the same. Was that just for information or were you proposing we remove `remove_padding`?
```r
> microbenchmark::microbenchmark(
+    dfm(tokens_remove(toks, "")),
+    dfm_remove(dfm(toks), ""),
+    dfm(toks, remove_padding = TRUE)
+ )
Unit: milliseconds
                             expr      min       lq     mean   median       uq      max neval
     dfm(tokens_remove(toks, "")) 24.52573 27.33274 32.28436 30.10545 33.06334 149.6006   100
        dfm_remove(dfm(toks), "") 24.52290 27.62597 33.32570 30.62006 32.69043 149.8663   100
 dfm(toks, remove_padding = TRUE) 22.91938 27.99522 32.07499 30.15143 32.86733 144.3347   100
```

Reply:
I misunderstood about the `valuetype = "fixed"` point earlier, thought you meant the deprecated `valuetype`, but you meant for `remove_padding`. Changed now in d54e6ea.